### PR TITLE
Test spot markets

### PR DESCRIPTION
**Pull request by noisekit**
# DAI
- [x] wrap fDAI -> to sDAI
- [x] sell sDAI -> for USDh
- [x] buy sDAI <- with USDh
- [x] unwrap fDAI <- from sDAI

# USDC
- [x] wrap USDC -> to sUSDC
- [x] sell sUSDC -> for USDh
- [x] buy sUSDC <- with USDh
- [x] unwrap USDC <- from sUSDC

### CODE REVIEW

File: e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_DAI.e2e.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+  it('should create user account', async () => {
+    assert.equal(
+      await getAccountOwner({ accountId }),
+      ethers.constants.AddressZero,
+      'New wallet should not have an account yet'
+    );
+    await createAccount({ wallet, accountId });
+    assert.equal(await getAccountOwner({ accountId }), address);
+  });
```
Comment (Tburm) [inline e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_DAI.e2e.js:83]: This shouldn't be required in this test

File: e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_USDC.e2e.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+  it('should create user account', async () => {
+    assert.equal(
+      await getAccountOwner({ accountId }),
+      ethers.constants.AddressZero,
+      'New wallet should not have an account yet'
+    );
+    await createAccount({ wallet, accountId });
+    assert.equal(await getAccountOwner({ accountId }), address);
+  });
```
Comment (Tburm) [inline e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_USDC.e2e.js:82]: this isn't necessary

File: e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_DAI.e2e.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    );
+  });
+
+  it('should swap 500 sDAI -> USDh', async () => {
+    assert.equal(await getCollateralBalance({ address, symbol: 'USDh' }), 0);
+    await spotSell({
+      wallet,
+      marketId: require('../../deployments/extras.json').synth_dai_market_id,
+      synthAmount: 500,
+      minUsdAmount: 400,
```
[the remark this answers is not in the document]
Comment (noisekit) [reply e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_DAI.e2e.js:135]: Yeah I thought of doing that. Maybe will update later

File: e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_DAI.e2e.js [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    );
+  });
+
+  it(`should unwrap 500 sDAI -> fDAI`, async () => {
+    const synthBalance = await unwrapCollateral({
+      wallet,
+      symbol: 'fDAI',
+      synthAddress: require('../../deployments/extras.json').synth_dai_token_address,
+      synthMarketId: require('../../deployments/extras.json').synth_dai_market_id,
+      amount: 500,
```
[the remark this answers is not in the document]
Comment (noisekit) [reply e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_DAI.e2e.js:191]: yeah as I was not able to get this call passing all follow up asserts were not correct. Thanks for updating

File: e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_USDC.e2e.js (hunk 2 above)
Comment (noisekit) [reply e2e/tests/omnibus-arbitrum-sepolia.toml/Spot_Market_USDC.e2e.js]: Indeed

**Pull request merged**